<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgb(230, 230, 250);*/
/*background-color: rgb(30, 144, 255);*/
background-color: #1A1A2A</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="userIdLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">/*color: rgb(255, 255, 255);*/
color: rgb(255, 255, 255); /* White text */
/*background-color: #1A1A2A /* Match window background */
padding: 5px; /* Padding for better readability */
font-family: Arial, sans-serif; /* Modern, clean font */
font-weight: bold; /* Bold text */
text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);</string>
         </property>
         <property name="text">
          <string>User Id</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>88</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="userIdLineEdit">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Please enter your user Id</string>
         </property>
         <property name="styleSheet">
          <string notr="true">/*background-color: rgb(158, 109, 97,0%);*/
 background-color: rgba(255, 255, 255, 0.8);
color: rgb(0, 0, 0);
border: 1px solid rgb(100, 100, 100);
 padding: 5px;
 border-radius: 5px;</string>
         </property>
         <property name="placeholderText">
          <string>Enter user Id</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="passwordLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">/*color: rgb(255, 255, 255);*/
/*color: rgb(255, 255, 255);*/
color: rgb(255, 255, 255); /* White text */
/*background-color: rgb(36, 121, 211); /* Match window background */
padding: 5px; /* Padding for better readability */
font-family: Arial, sans-serif; /* Modern, clean font */
font-weight: bold; /* Bold text */
text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);</string>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Please enter your Password</string>
         </property>
         <property name="styleSheet">
          <string notr="true"> background-color: rgba(255, 255, 255, 0.8);
color: rgb(0, 0, 0);
border: 1px solid rgb(100, 100, 100);
 padding: 5px;
 border-radius: 5px;</string>
         </property>
         <property name="placeholderText">
          <string>Enter Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-color: rgb(135, 206, 250);*/
/*background-color: rgb(50, 205, 50);*/
/*background-color: rgb(255, 140, 0);*/
/*background-color: rgb(42, 42, 74);*/
/*background-color: rgb(26, 74, 74);*/
/*background-color: rgb(26, 42, 74);*/
/*background-color: rgb(74, 74, 42);*/
/*background-color: rgb(74, 74, 90);*/
/*background-color: rgb(42, 42, 58);*/
/*background-color: rgb(0, 255, 255);*/
background-color: rgb(255, 105, 180);
/*background-color: rgb(26, 26, 42);*/
/*background-color: rgb(255, 140, 0);*/</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>7</y>
       <width>521</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Cooper Black</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">/*color: rgb(75, 0, 130);*/
/*color: rgb(138, 43, 226);*/
/*color: rgb(255, 193, 7);*/
/*color: rgb(64, 0, 255);*/
color: rgb(0, 0, 0);
/*color: rgb(255, 215, 0);*/</string>
        </property>
        <property name="text">
         <string>Login Window</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="forgotPasswordButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>179</width>
      <height>47</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(50, 205, 50); /* Green background */
    color: rgb(0, 0, 0); /* White text */
    border: 2px solid rgb(34, 139, 34); /* Darker green border */
    padding: 10px 20px; /* Padding for better appearance */
    border-radius: 5px; /* Rounded corners for a modern look */
    font-size: 16px; /* Adjust font size */
    font-weight: bold; /* Bold text */
    transition: background-color 0.3s ease, border-color 0.3s ease; /* Smooth transition */
}

QPushButton:hover {
    background-color: rgb(34, 139, 34); /* Darker green on hover */
}

QPushButton:focus {
    border-color: rgb(255, 255, 255); /* White border on focus */
    box-shadow: 0 0 5px rgba(255, 255, 255, 0.5); /* Optional shadow on focus */
}
</string>
    </property>
    <property name="text">
     <string>Forgot Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newUserButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);
background-color: rgb(85, 85, 255);</string>
    </property>
    <property name="text">
     <string>New User </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Loginbutton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>86</width>
      <height>47</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(50, 205, 50); /* Green background */
    color: rgb(0, 0, 0); /* White text */
    border: 2px solid rgb(34, 139, 34); /* Darker green border */
    padding: 10px 20px; /* Padding for better appearance */
    border-radius: 5px; /* Rounded corners for a modern look */
    font-size: 16px; /* Adjust font size */
    font-weight: bold; /* Bold text */
    transition: background-color 0.3s ease, border-color 0.3s ease; /* Smooth transition */
}

QPushButton:hover {
    background-color: rgb(34, 139, 34); /* Darker green on hover */
}

QPushButton:focus {
    border-color: rgb(255, 255, 255); /* White border on focus */
    box-shadow: 0 0 5px rgba(255, 255, 255, 0.5); /* Optional shadow on focus */
}
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>userIdLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>Loginbutton</tabstop>
  <tabstop>forgotPasswordButton</tabstop>
  <tabstop>newUserButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
